dusk-labyrinth.lic

```diff
@@ -7,17 +7,15 @@ custom_require.call %w[common common-items events]
 class DuskLab
   include DRC
   include DRCI
-  def initialize
-    arg_definitions = [
-      [{ name: 'loot_bag', display: 'container noun', regex: /\w+/i, description: 'Bag for your loot!' },
-       { name: 'redeem_scrip', display: 'redeem', regex: /^redeem/i, optional: true, description: 'Redeem scrip when finished?' }]
-    ]
-    args = parse_args(arg_definitions)
+  def initialize(settings)
+    @labyrinth_data = settings.duskruin['labyrinth']
```

Comment:
`base.yaml` needs to have a copy of these settings' defaults, and potentially `base-empty.yaml` depending on their data type

Reply:
Sure thing! Changes made per https://github.com/rpherbig/dr-scripts/pull/4390/commits/7344daf1d596af1bdd6069a7091b3df540559aac